stable/gcloud-sqlproxy/templates/networkpolicy.yaml

```diff
@@ -0,0 +1,24 @@
+{{- if .Values.networkPolicy.enabled }}
+kind: NetworkPolicy
+apiVersion: networking.k8s.io/v1
+metadata:
+  name: "{{ include "gcloud-sqlproxy.fullname" . }}"
+  labels:
+    app: {{ include "gcloud-sqlproxy.name" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+spec:
+  podSelector:
+    matchLabels:
+      app: {{ include "gcloud-sqlproxy.name" . }}
```

Comment:
make `matchLabels:` configurable as well

Reply:
This `matchlabels` needs to match what is in the deployment template so it can target the gcloud-sql deployment, I think it is fine, please double check.

Reply:
👍